Nimb3s.Automaton.DataAccess/JobData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Nimb3s.Automaton.Pocos;
+using Nimb3s.Automaton.Pocos.Models;
```

Comment:
you will want to create a new entity that maps to the columns you want to return. in your case you will want to  create an entity like this and place it in [`Nimb3s.Automaton.Core`](https://github.com/nimb3s/automaton/tree/main/src/Nimb3s.Automaton.Core). later on will move all those entities into their own lib called Nimb3s.Automaton.Core.Entties:

```csharp
public class JobStatusDetailsEntity : IEntity<Guid>
{
 public Guid Id {get;set;} //the jobStatusId
 public Guid JobId {get;set;}
 public string JobName {get;set;}
 public short JobStatusId {get;set;}
}
```
you can take the  [`WorkItemStatusRepositroy`](https://github.com/nimb3s/automaton/blob/31e5b228cf5fee8ebee861ff6b5fc8c0caa61f9f/src/Nimb3s.Automaton.Core.Repositories.Sql/WorkItemStatusRepository.cs#L20) as an example.